rbac/server/api/roles.py

```diff
@@ -266,6 +422,23 @@
 
 
 @ROLES_BP.post("api/roles/<role_id>/admins")
+@doc.summary("Add an admin to the role.")
+@doc.description("Add an admin to the role.")
```

Comment:
For the add admin/owner/member it actually creates a proposal for the role owner to approve. We should probably note this in the descriptions for these three endpoints.

Reply:
Also needs to be added to the add role owner and add role member endpoints.